src/lib.rs

```diff
@@ -75,9 +77,32 @@ pub fn sealed(_args: TokenStream, input: TokenStream) -> TokenStream {
     })
 }
 
+// generously offered by @danielhenrymantilla aka Yandros
+fn to_snake_case(s: &'_ str) -> String {
```

Comment:
You may omit `'_` lifetime here. Lifetime elision rules will do everything as expected in this situation. And [`elided_lifetimes_in_paths` rustc lint](https://doc.rust-lang.org/rustc/lints/listing/allowed-by-default.html#elided-lifetimes-in-paths) doesn't apply to to references, it's mandates only lifetime parameters in angle brackets.